.github/workflows/send-transactions-test.yml

```diff
@@ -0,0 +1,586 @@
+name: Test commit
+
+on:
+  pull_request: {}
```

Comment:
do we want to run it automatically on every push to pull requests? It may be time consuming and may block the pull request for a long time (so I'd recommend to turn it off as a required check). IMO it may be worth considering running it manually (then there is a button to manually trigger the action) or using some cron action (once a day for example).

Reply:
It was informed to me that this should behave the same as our current action that checks the code. Could we sync this on our meeting to see if we need to change. It takes around 15min to run completely it's not absolutely high but takes some time.

Reply:
Sure, please add that to our meeting notes for tomorrow